virt/virt-2-5-release-notes.adoc

```diff
@@ -149,6 +149,8 @@ The *Pending Changes* banner at the top of the page displays a list of all chang
 
 * When nodes are evicted, for example, when they are placed in maintenance mode during an {product-title} cluster upgrade, virtual machines are migrated twice instead of just once. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1888790[*BZ#1888790*])
 
+* Following upgrade, there may be more than one template per operating system workload. When creating a Windows virtual machine from a cloned PVC (default OS image), the *OS* must have the correct *Workload* defined. A template is defined with the following GUI labels: *OS*, *Flavor*, and *Workload*. On Microsoft Windows, for example, selecting a *Workload* value of *server* does not allow you to use a default OS image, even though the *(Source available)* label displays in the GUI. The default OS image image is attached to the newer template but the wizard may use the old template, which is not configured to support default OS images. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1907183[*BZ#1907183*])
```

Comment:
* Following upgrade -> Following an upgrade?
* may -> might (two instances of this)
* Typically you don't want to shorten operating system to OS. So unless this is something you're regularly doing in the CNV docs, I would spell it out. See [1].
* I get that this references items from the GUI, but I think that there is too much bolded here. I would stick to only bolding whenever you're really referring to a field in the GUI. For example, I would at least not bold anything in the second sentence.
* Should "GUI" be "web console" here? (two instances)
* Double "image image"
* Not critical if you want to keep as is, but the sentence about telling which labels are used seems out of place, just questioning if it's necessary.

[1] https://redhat-documentation.github.io/supplementary-style-guide/#operating-system

Reply:
Hi Andrea,

In this case the feature from last release was named "default OS images". Changing that I fear will impact find-ability, but I can spell out OS on the first occurrence. 

The to bold or not to bold is very confusing to me. It seems very subjective. But I can remove the bolding from the second sentence.

Ready for re-review.

Reply:
That's totally fine then. However when you spell things out, normally you put the abbreviation in parentheses, not the other way around. You do already spell out operating system in the first sentence, so I'd suggest just putting (OS) after that there instead of the (operating system) you added in the second sentence.

So:

```
Following an upgrade, there might be more than one template per operating system (OS) workload. When creating a Microsoft Windows virtual machine from a cloned PVC using the default OS images feature...
```

wdyt?